src/shared/modules/user-favorites/user-favorites.duck.js

```diff
@@ -0,0 +1,281 @@
+/*
```

Comment:
Could you follow the duck file naming convention we already use here?
I like your naming, but we should be consistent.

Reply:
the reason i went use kebab-case is that it's generally recommended to use lowercase names. I'll update it :)

Reply:
I agree, but we should make one big effort to rename all files at the same time.